frontend/templates/troubleshooting/index.tsx

```diff
@@ -84,6 +85,13 @@ const Wiki: NextPageWithLayout<{
       </>
    );
 
+   const imageSx: any = {
```

Comment:
I think you can import `SystemProps` from chakra and that will add the type

Reply:
I tried both `SystemProps` and `SystemStyleProps` and got this error:
```
Element implicitly has an 'any' type because expression of type '"@media (min-width: sm)"' can't be used to index type 'SystemProps'.
  Property '@media (min-width: sm)' does not exist on type 'SystemProps'.ts(7053)
```